Code/Editor/CryEditDoc.cpp

```diff
@@ -1108,7 +1108,7 @@ bool CCryEditDoc::SaveLevel(const QString& filename)
     if (QFileInfo(filename).isRelative())
     {
         // Resolving the path through resolvepath would normalize and lowcase it, and in this case, we don't want that.
-        fullPathName = Path::ToUnixPath(QDir(QString::fromUtf8(gEnv->pFileIO->GetAlias("@devassets@"))).absoluteFilePath(fullPathName));
+        fullPathName = Path::ToUnixPath(QDir(QString::fromUtf8(gEnv->pFileIO->GetAlias("@projectroot@"))).absoluteFilePath(fullPathName));
```

Comment:
Well currently the `<project-root>` is still the monitored folder assets within a project instead of `<project-root>/Assets` in order to maintain backwards compatibility with existing projects.

The Scattered assets in the current existing projects were never moved to the "Assets" folder.

Reply:
With the updates to the level system the prefabs that now store the level data can be placed anywhere, though the original convention of <project>/Levels is hard to break and is still where people typically store their prefabs with level data there.